src/pages/documentation/faq.md

```diff
@@ -111,6 +111,52 @@ Add the following rule to your custom ProGuard rules file, typically labeled `pr
 }
 ```
 
+### Is there a change in minimum API level supported by Android SDK?
+
+Android SDK now supports a minimum API level of **19**.  If your application targets a lower API level, you will see the following build failure:
+
+```
+Manifest merger failed : uses-sdk:minSdkVersion 14 cannot be smaller than version 19 declared in library [com.adobe.marketing.mobile:core:2.0.0]
+```
+
+To fix this build failure, increase the minSdkVersion for your Android project to **19** or above.
+
+### When I add Android SDK to my project, why do I get an error about invoke-custom support and enabling desugaring?
+
+Android SDK uses Java 8 language features and desugaring is disabled by default. If your application uses Android Gradle plugin (AGP) v4.2 and has not enabled Java 8 support, you will see the following build failure:
+
+```
+D8: Invoke-customs are only supported starting with Android O (--min-api 26)
+Caused by: com.android.builder.dexing.DexArchiveBuilderException: Error while dexing.
+The dependency contains Java 8 bytecode. Please enable desugaring by adding the following to build.gradle
+android {
+	compileOptions {
+		sourceCompatibility 1.8
+		targetCompatibility 1.8
+	}
+}
+
+See https://developer.android.com/studio/write/java8-support.html for details.
+Alternatively, increase the minSdkVersion to 26 or above.
+```
+
+To fix this build failure, you can follow one of two options:
+- Add the listed compileOptions from the error message to your app-level build.gradle file.
+- Increase the minSdkVersion for your Android project to 26 or above.
+
+### Why do I see a warning in AndroidManifest.xml about missing 'com.adobe.marketing.mobile.FullscreenMessageActivity' class?
+
+After upgrading to latest Android SDK, you will see the following build warning if your application previously set up in-app messages with Campaign Standard. 
```

Comment:
"... upgrading to the latest version of the Mobile SDK for Android, you will..."